Cargo.toml

```diff
@@ -28,8 +28,9 @@ members = [
     "rsa",
     "ecdsa",
     "p256",
-    "blake2",
+    "blake2", 
 ]
+exclude = ["fstar-helpers/minicore"]
```

Comment:
Why do you want to exclude it from the workspace?

Reply:
I can include it, it felt like it was nicer to exclude